ZendeskApi_v2/Requests/Attachments.cs

```diff
@@ -87,33 +88,56 @@ public Upload UploadAttachments(IEnumerable<ZenFile> files)
         Upload UploadAttachment(ZenFile file, string token, int? timeout)
```

Comment:
RunRequest takes different parameters and can't handle filedata as it is right now. Do you want Core.RunRequest to be refactored or to make a new method?